src/pykiso/exceptions.py

```diff
@@ -14,12 +14,14 @@
 
 .. currentmodule:: exceptions
 """
+import logging
 
 
 class PykisoError(Exception):
     """Pykiso specific exception used as basis for all others."""
 
     def __str__(self):
+        logging.exception(self.message)
```

Comment:
As discussed don't mix up logging and exceptions, just add the log.exception when the exceptions are caught

Reply:
Ok, right just wanted to display the full message, as I didn't know it will be anyway